release_notes/ReleaseNotes_v11.0.0.md

```diff
@@ -111,6 +111,8 @@ See [here](https://github.com/ISISComputingGroup/IBEX/wiki#instrument-informatio
 | [#7230](https://github.com/ISISComputingGroup/IBEX/issues/7230) | Minor | Removed "add block to a config" (not current config) option on right-click - see https://github.com/ISISComputingGroup/IBEX/issues/7224 |
 | [#7236](https://github.com/ISISComputingGroup/IBEX/issues/7236) | Minor | Added the ability to search IOCs when adding to configuration and in Start/Stop IOCs dialog |
 | [#7229](https://github.com/ISISComputingGroup/IBEX/issues/7229) | Minor | Use https links in GUI help pages |
+| [#7269](https://github.com/ISISComputingGroup/IBEX/issues/7269) | Minor | Fix "edit host configuration/component" button in block dashboard menu |
```

Comment:
This is probably "patch" as the functionality previously existed, and we just introduced & then fixed a bug between releases?